test/classes/claridge/baseConstructorCall.chpl

```diff
@@ -1,17 +1,18 @@
 class BaseClass {
   var a, b: int;
 
-  proc BaseClass(b: int) {
+  proc init(b: int) {
     this.b = b;
+    super.init();
```

Comment:
In rewriting existing tests, I'd be inclined to take the stance of only using super.init() when it's necessary since it's more likely to be what a typical user does and may lend fodder to any debates around switching the default constructor phase going forward.  I'd also keep a list of cases where it seemed regrettable that the default wasn't the other way around (where double-initialization in a case like this is one such case).